collectors/python.d.plugin/haproxy/metadata.yaml

```diff
@@ -26,29 +29,90 @@ overview:
-    list: []
+    list:
+      - title: 'HAProxy setup for socket'
+        description: 'Socket must be readable and writable by the netdata user.'
+      - title: 'HAProxy setup for URL'
+        description: 'URL must have `stats uri <path>` present in the haproxy config, otherwise you will get HTTP 503 in the haproxy logs.'
   configuration:
     file:
-      name: ''
-      description: ''
+      name: python.d/haproxy.conf
     options:
-      description: ''
+      description: |
+        There are 2 sections:
+
+        * Global variables
+        * One or more JOBS that can define multiple different instances to monitor.
+
+        The following options can be defined globally: `priority`, `penalty`, `autodetection_retry`, `update_every`, but can also be defined per JOB to override the global values.
+
+        Additionally, the haproxy plugin also supports the following: `user`, `pass`, `url` and `socket` inside a JOB definition.
+
+        Every configuration JOB starts with a `job_name` value which will appear in the dashboard, unless a `name` parameter is specified.
       folding:
-        title: ''
+        title: "Config options"
         enabled: true
-      list: []
+      list:
+        - name: update_every
+          description: Sets the default data collection frequency.
+          default_value: 5
+          required: false
+        - name: priority
+          description: Controls the order of charts at the netdata dashboard.
+          default_value: 60000
+          required: false
+        - name: autodetection_retry
+          description: Sets the job re-check interval in seconds.
+          default_value: 0
+          required: false
+        - name: penalty
+          description: Indicates whether to apply penalty to update_every in case of failures.
+          default_value: yes
+          required: false
+        - name: name
+          description: Job name. This value will overwrite the `job_name` value. JOBS with the same name are mutually exclusive. Only one of them will be allowed running at any time. This allows autodetection to try several alternatives and pick the one that works.
+          default_value: ''
+          required: false
+        - name: user
+          description: Username if stats auth is used.
+          default_value: ''
+          required: false
+        - name: pass
+          description: Password if stats auth is used.
+          default_value: ''
+          required: false
+        - name: url
+          description: URL to the haproxy_stats endpoint. Also make sure the parameters `csv` and `norefresh` are provided.
+          default_value: ''
+          required: false
+        - name: socket
+          description: Unix socket path to the haproxy sock file.
+          default_value: ''
+          required: false
     examples:
       folding:
         enabled: true
-        title: ''
-      list: []
+        title: "Config"
+      list:
+        - name: URL method
```

Comment:
you can additionally always set `folding: false` to the first configuration example